server/form-pages/assess/matchingInformation/matchingInformationTask/matchingInformation.ts

```diff
@@ -1,20 +1,18 @@
+import type { ApType, Gender } from '@approved-premises/api'
 import type { TaskListErrors } from '@approved-premises/ui'
 
 import { Page } from '../../../utils/decorators'
 
 import TasklistPage from '../../../tasklistPage'
 import { lowerCase, sentenceCase } from '../../../../utils/utils'
 
-const apTypes = {
-  standard: 'Standard AP',
+const apTypes: Record<ApType, string> = {
+  normal: 'Standard AP',
   pipe: 'Psychologically Informed Planned Environment (PIPE)',
   esap: 'Enhanced Security AP (ESAP)',
-  rfap: 'Recovery Focused Approved Premises (RFAP)',
```

Comment:
Ah, OK. I'll update the API spec in that case. Think it makes sense 👍

Reply:
I would hold fire until Beth can confirm as I may have misunderstood

Reply:
To be fair, I was being lazy here, thought we could always put it back in

Reply:
We originally had it in here, as the user will need to specify if RFAP is required. I caught up with Dan + Sarah this morning though and there was a discussion around whether we need it here on the placement requirements list. There's some design work to be done as we think it will sit with the 'Semi-specialist mental health' option now. 
I'm going to have a look at the design work tomorrow, so can send it over to you as soon as it's done.